framework/console/controllers/AssetController.php

```diff
@@ -326,6 +327,11 @@ protected function loadTargets($targets, $bundles)
     protected function buildTarget($target, $type, $bundles)
     {
         $tempFile = $target->basePath . '/' . strtr($target->$type, ['{hash}' => 'temp']);
+        if (!is_file($tempFile)) {
+            FileHelper::createDirectory(dirname($tempFile), 0755, true);
+            touch($tempFile);
```

Comment:
I still have this issue.
But had no time to investigate the additional check yet.
Please leave it open for a few more days.